packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.0]
+
+### Added
+
+- Now exports the `Domain` type ([#4104](https://github.com/MetaMask/core/pull/4104))
+
+### Changed
+
+- **BREAKING:** `SelectedNetworkController` now expects a `domainProxyMap` param - which is a Map of Domain to NetworkProxy - in its constructor ([#4104](https://github.com/MetaMask/core/pull/4104))
+- **BREAKING:** `SelectedNetworkController` now requires the `NetworkController:getSelectedNetworkClient` as an allowed action ([#4063](https://github.com/MetaMask/core/pull/4063))
+- `getProviderAndBlockTracker` method no longer throws an error if the `useRequestQueue` flag is false
+- Previously `getProviderAndBlockTracker` threw an error if there was no `networkClientId` set for the passed domain. Now it returns a proxy pointed at the globally selected network instead.
```

Comment:
Nit: Perhaps we could describe the purpose of this change (seamlessly proxying calls during sessions where a dapp connects/disconnects), and group these changes under it? That would provide a better explanation for why we've made these changes. In isolation these changes seem a bit arbitrary.

Reply:
done here: https://github.com/MetaMask/core/pull/4111/commits/6211e4e2637f4d18ff1c56d4180e8f0b7d999477